README.md

```diff
@@ -28,25 +28,29 @@ The current multibase table is [here](multibase.csv):
 
 ```
 encoding      codes   name
-base1         1       unary tends to be 11111
-base2         0       binary has 1 and 0
-base8         7       highest char in octal
-base10        9       highest char in decimal
-base16        F, f    highest char in hex
-base32        U, u    rfc4648 - highest letter
-base32hex     V, v    rfc4648 - highest char
-base58flickr  Z       highest char
-base58btc     z       highest char
-base64        y       rfc4648 highest char
-base64url     Y       rfc4648 highest char
-binary        b       8-bit binary (encoder and decoder keeps data unmodified)
+base1,        1,    unary tends to be 11111
+base2,        0,    binary has 1 and 0
+base8,        7,    highest char in octal
+base10,       9,    highest char in decimal
+base16,       F, f, highest char in hex
+base32hex,    V, v, rfc4648 no padding - highest char
+base32hexpad, T, t, rfc4648 with padding
+base32,       B, b, rfc4648 no padding
+base32pad,    C, c, rfc4648 with padding
+base32z,      h,    z-base-32 - used by Tahoe-LAFS - highest letter
+base58flickr, Z,    highest letter
+base58btc,    z,    highest letter
+base64,       m,    rfc4648 no padding
+base64pad,    M,    rfc4648 with padding - MIME encoding
+base64url,    u,    rfc4648 no padding
+base64urlpad, U,    rfc4648 with padding
+binary,       X,    8-bit binary (encoder and decoder keeps data unmodified)
```

Comment:
I think _binary_ as you have it, is the "zero encoding" or "identity encoding" and thus should be tagged with `0x0000`. it does not need to be human readable because the rest of the string wont be. It's also easier to understand at first glance.